src/__main__.py

```diff
@@ -1,55 +1,30 @@
-
-from corrigir import corrigir
-from geradores.gerar_json_aluno import gerar_json_alunos
-from geradores.gerar_json_geral_disciplina import gerar_json_disciplinas
-from leitura_e_escrita.escrever_arquivo import escrever_csv, escrever_json
-from leitura_e_escrita.ler_arquivo import csvs_to_dfs
+from corretores import corrigir
+from geradores import gerar_relatório
+from leitura_e_escrita.ler_arquivo import carregar_dados
+from src.auxilio.checker import check
 from gui import Aplication
 from auxilio.path import (get_caminho_de_saida, join_paths,
                           ROOT_PATH)
 
 
-def main(dados):
+def main(dados: dict):
 
 
-    dados_alunos_path = dados["caminhos_dados"][0] # so um item por enquanto, por isso [0]
-    respostas_alunos_path = dados["caminhos_respostas"][0]
-    gabarito_path = dados["caminhos_gabaritos"][0]
+    dir_de_entrada = dados["dir_entrada"]
+    dir_saida = get_caminho_de_saida(dados["dir_saida"])
     tipo_correcao = dados["tipo_de_correcao"]
 
-    path_output_arquivos_correcao = get_caminho_de_saida(dados["caminho_de_saida"])
+    # Carregar dados
+    dados_de_entrada = carregar_dados(dir_de_entrada, tipo_correcao)
 
-    _, df_respostas, df_gabarito = csvs_to_dfs(
-        dados_alunos_path, respostas_alunos_path, gabarito_path
-    )
-    df_redacoes = []  # PLACE HOLDER Não implementado
+    #checar se os dados são válidos
```

Comment:
```suggestion
    # Checar se os dados são válidos
```

Afim de manter o padrão, os comentários no nosso código tem um espaço e começam com letra maiuscula